modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1071,6 +1079,16 @@ public abstract class Base${schemaName}ResourceImpl
 		protected UnsafeBiConsumer<Collection<${javaDataType}>, UnsafeConsumer<${javaDataType}, Exception>, Exception> contextBatchUnsafeConsumer;
 	</#if>
 
+	<#if generateUpdateExternalReferenceCode>
+		protected ${schemaName} updateExternalReferenceCode(
+					String externalReferenceCode,
+					${properties["id"]} ${schemaVarName}Id)
```

Comment:
I am not sure what properties contains in this case :thinking: I think it could be overriden, right?